lte/gateway/python/magma/pipelined/gtp_stats_collector.py

```diff
@@ -144,7 +144,7 @@ def match_gtp_lines(lines):
         return create_error_result(stderr)
 
     try:
-        stdout_lines = stdout.decode('ascii').split('\n')
```

Comment:
Are you sure this can be removed? The method signature claims stdout is a string but did you check that the caller really passes a string?

Reply:
Apparently this does not get tested in the unit / sudo tests. How should we proceed? Change `str` to `bytes` instead or keep my first version?

Reply:
Change `str` to `bytes` (probably for both inputs).

Assuming that this code worked before (even if it is untested), the change would break it.

Reply:
The `stderr` is parsed into the `OVSDBCommandResult` NamedTuple, typed there as an `Optional[str]`.

Reply:
You're right, that's really strange. So only `stdout` seems to be bytes, `stderr` seems to be a string...